### DISCUSSION TITLE
请尽快办理退款

### DISCUSSION THREAD

**Discussion by David68326** [getlantern/lantern-forum-cn#529] [General]
@Mike777Khoo 
之前说的14天左右可以帮我办理退款，现在已经快一个月了，请尽快办理一下，谢谢！

**Comment by MaxMKMax** [comment]
@David68326 您好. 请问您是在哪个帖子提交问题的呢? 您方便重新给我支付单据和账号邮箱吗? 谢谢.

**Reply by David68326** [reply]
@Mike777Khoo #465 ,我在原帖子上又回复了您一遍
